sdk/v1/client.go

```diff
@@ -68,32 +68,35 @@ func (cli *Client) Checker(ctx context.Context, check *health.CheckState) error
 }
 
 // PostJob creates a new reindex job for processing
-func (cli *Client) PostJob(ctx context.Context, headers client.Headers) (*models.Job, error) {
+func (cli *Client) PostJob(ctx context.Context, reqHeaders client.Headers) (*client.RespHeaders, *models.Job, error) {
 	var job models.Job
-	if headers.ServiceAuthToken == "" {
-		headers.ServiceAuthToken = cli.serviceToken
+	if reqHeaders.ServiceAuthToken == "" {
+		reqHeaders.ServiceAuthToken = cli.serviceToken
 	}
 
 	path := cli.hcCli.URL + "/" + cli.apiVersion + jobsEndpoint
```

Comment:
```Suggestion is to use
	path := fmt.Sprintf("%s/%s/jobsEndpoint", cli.hcCli.URL, cli.apiVersion)
```

Reply:
I'm happy to change this but then I think I should also change it for PostTask, PatchJob, and PutJobNumberOfTasks?

Reply:
done